development_scripts.py

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env python
```

Comment:
I think we should add tests for this as well since it is a utility we're providing and want to make sure everything works in it as intended.

Reply:
I did refactor the functions last night with this in mind. Also need to add tests for `lib/ntc-templates/parse.py`